src/testplan/testplanTestRunner.ts

```diff
@@ -0,0 +1,225 @@
+import * as path from 'path';
+import * as tmp from 'tmp';
+import {
+    TestEvent, TestSuiteInfo
+} from 'vscode-test-adapter-api';
+
+import { ArgumentParser } from 'argparse';
+import { IWorkspaceConfiguration } from '../configuration/workspaceConfiguration';
+import { IEnvironmentVariables, EnvironmentVariablesLoader } from '../environmentVariablesLoader';
+import { ILogger } from '../logging/logger';
+import { IProcessExecution, runProcess } from '../processRunner'
+import { IDebugConfiguration, ITestRunner } from '../testRunner';
+import { empty } from '../utilities/collections';
+import { setDescriptionForEqualLabels } from '../utilities/tests';
+import { parseTestStates } from './testplanJunitTestStatesParser';
+import { parseTestSuites } from './testplanTestCollectionParser';
+
+// --- Testplan Exit Codes ---
+// 0: All tests were collected and passed successfully
+// 1: Tests were collected and run but some of the tests failed or none found
+const TESTPLAN_NON_ERROR_EXIT_CODES = [0, 1];
```

Comment:
I think this can be removed. This option was used only for pytest script wrappers.

Reply:
We need the errno 2 for the described scenario above.